kategory/src/main/kotlin/kategory/data/ListKW.kt

```diff
@@ -0,0 +1,49 @@
+package kategory
+
+typealias ListKindW<A> = HK<ListKW.F, A>
+
+fun <A> ListKindW<A>.ev(): ListKW<A> =
+        this as ListKW<A>
+
+class ListKW<A> private constructor(private val list: List<A>) : ListKindW<A>, Collection<A> by list {
+
+    class F private constructor()
+
+    fun <B> map(f: (A) -> B): ListKW<B> = ListKW(list.map(f))
+
+    fun <B> flatMap(f: (A) -> ListKW<B>): ListKW<B> = ListKW(list.flatMap { f(it).list })
+
+    operator fun plus(list: List<@UnsafeVariance A>): ListKW<A> = ListKW(this.list + list)
```

Comment:
Was this unsafe in List too? and why is it unsafe? Perhaps these can be moved to extension functions.